src/main/java/dev/ebrydeu/spring_boot_library/domain/entities/User.java

```diff
@@ -0,0 +1,24 @@
+package dev.ebrydeu.spring_boot_library.domain.entities;
+
+import lombok.*;
+import jakarta.persistence.*;
+import java.util.UUID;
+@Data
+@Entity
+@Table(name = "USER")
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @NonNull private UUID id;
+
+    @NonNull private String firstName;
+    @NonNull private String lastName;
+    @NonNull private String profileName;
+    @NonNull private String profilePicture;
+    @NonNull private String email;
```

Comment:
Please replicate the like in Message entity and also ensure that some fields are mapped  to MySQL convention. This can be achieved using `@Column(name = "profile_name")`.